src/taipy/core/job/_job_manager.py

```diff
@@ -53,7 +52,7 @@ def _create(
             version=version,
         )
         cls._set(job)
-        _publish_event(cls._EVENT_ENTITY_TYPE, job.id, EventOperation.CREATION, None)
+        _publish_event(EventEntityType.JOB, EventOperation.CREATION, entity_id=job.id, config_id=job._task.config_id)
```

Comment:
I would not pass the task configuration identifier as the `config_id` parameter for a job event. This looks strange to me. I would put it in the metadata. 
`config_id` is used differently for other entities. It corresponds to the actual configuration object used to generate the entity. It is different for the jobs.

Is there any issue with naming the event attribute `task_config_id`?

Reply:
We can pass it to medata if it make more sense.
And yes we can name it `task_config_id`.

However, for a scenario or a data node, does it make sense NOT to give the "config ID"? How do we know which scenario or datanode we are talking about?

Reply:
Scenarios and datanodes depend on a configuration so they have a `config_id` attribute.
It does NOT make sense not to provide the config ID. It must be passed.

Jobs don't depend on a configuration. They depend on a task that depends on a configuration.
What you want to provide here is not the `config_id` of the job (it does not exist) but the `config_id` of the task attached to the job.

Does it make sens to you?

Reply:
Yes, it does !
Thank you for your feedback!